packages/contracts-bedrock/src/L2/OptimismSuperchainERC20.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+// import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { ERC20 } from "@solady/tokens/ERC20.sol";
+import { IL2ToL2CrossDomainMessenger } from "src/L2/IL2ToL2CrossDomainMessenger.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+import { SafeCall } from "src/libraries/SafeCall.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+/// @notice Thrown when attempting to relay a message and the function caller (msg.sender) is not
+/// L2ToL2CrossDomainMessenger.
+error RelayMessageCallerNotL2ToL2CrossDomainMessenger();
```

Comment:
What do you think about making the error messages shorter:
1. CallerNotL2ToL2CrossDomainMessenger for this one 
2. InvalidCrossDomainSender or InvalidSender for the MessageSenderNotThisSuperchainERC20
3. OnlyBridge instead of CallerNotBridge

They are a bit less explicit but shorter, which make the code a bit cleaner